.babelrc

```diff
@@ -1,5 +1,5 @@
 {
-    "presets": ["stage-0", "react"],
+    "presets": ["stage-0", "react", "es2015"],
```

Comment:
I have checked this on the v25.0.x branch and the lib being generated is looking fine.